src/libratbag.h

```diff
@@ -1125,6 +1141,12 @@ enum ratbag_button_type {
 	RATBAG_BUTTON_TYPE_PROFILE_DOWN,
 };
 
+struct ratbag_color {
+	unsigned short red;
+	unsigned short green;
+	unsigned short blue;
+};
```

Comment:
I'd rather go for unsigned int, as @whot, or even a double (normalized between 0 and 1.0). The reason being that in the same way we are seeing 10-bits per color channel capable monitor, I wouldn't be surprised if some HW manufacturer decide to have more color capable LEDs.